src/vgspec.jl

```diff
@@ -25,7 +29,6 @@ function (p::VGSpec)(data, name::String)
 
     it = IteratorInterfaceExtensions.getiterator(data)
     vg_set_spec_data!(new_dict, it, name)
-    detect_encoding_type!(new_dict, it)
```

Comment:
@mcmcgrath13 I believe you added that line originally. I don't think that ever did anything, right? Vega doesn't even have the encoding field, as far as I can tell, so I think this removal should be harmless? If not, let me know!